### PR TITLE
feat(ui): Add Loop, PIP, Cast, AirPlay buttons to control panel

### PR DESCRIPTION
**Pull request by nbcl**
## Description

The present pull request provides a partial solution to #2676 (Add overflowMenuButtons to the UI bar) by making the Picture-in-picture, Loop, Cast and AirPlay buttons available in the UI control panel.

The final result is achieved using the [CSS display none approach](https://github.com/google/shaka-player/issues/2676#issuecomment-801642557) suggested by @joeyparrish, and looks as follows:

<p align=center>

</p>

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change includes a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [x] I have made the corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by joeyparrish** [comment]
> To correctly follow the user experience intentions of Shaka Player, I would like to know which pair combination of the following icons would be ideal to represent the "loop on" and "loop off" states, as there is no "loop_off" icon in the library.

How about "repeat" and "repeat_on" ?

image

File: ui/less/containers.less [hunk 1]
```diff
@@ -170,6 +170,11 @@
   }
 }
 
+/* Buttons hide certain items if they are found inside the control panel */
+.shaka-controls-button-panel .display-none-if-on-control {
```
Comment (joeyparrish) [inline ui/less/containers.less:174]: This class name feels weird to me.  I think "shaka-overflow-menu-only" might be a better name, but I'd be happy to debate it if you disagree.

As the saying goes, there's really only one hard problem in computer science: naming things, and off-by-one errors.  :grin:

**Review by joeyparrish — COMMENTED** [review]
It looks great otherwise.  I really only had this one complaint.  Nice job!

File: ui/less/containers.less (hunk 1 above)
Comment (nbcl) [reply ui/less/containers.less]: I am ashamed to admit how much time it took me to understand that saying the first time I read it. 

Oh well, I changed it now, let us see what other weird variable names I come up with next. 😄

**Comment by nbcl** [comment]
Alright, I am marking this pull request as ready for review! 🎉

**Pull request merged**